blog/content/post/2020-03-27-trenchboot-nlnet-lz.md

```diff
@@ -0,0 +1,419 @@
+---
+title: Open Source DRTM with TrenchBoot. Project's basics.
+abstract: Today, we will get you into project structure and environment. You
```

Comment:
We don't know when someone will read this post. I would suggest to change abstract to something like this:

```
This is the first blog post from the Trenchboot series. It will introduce you to the project, its
structure and environment. Additionally the reader will find out more about each project component,
how to setup the environment and configure the build. 
```

Reply:
Done